ramls/inventory-hierarchy/inventory-instance-records.json

```diff
@@ -115,6 +115,32 @@
                     "type": "string"
                   }
                 }
+              },
+              "effectiveLocation": {
+                "type": "object",
+                "description": "Effective location, shelving location, or holding which is a physical place where items are stored, or an Online location",
+                "properties": {
+                  "name": {
+                    "description": "Name of the (shelf) location",
+                    "type": "string"
+                  },
+                  "campusName": {
+                    "description": "The name of the campus, the second-level location unit, this (shelf) location belongs to",
+                    "type": "string"
+                  },
+                  "libraryName": {
+                    "description": "The name of the library, the third-level location unit, this (shelf) location belongs to",
+                    "type": "string"
+                  },
+                  "institutionName": {
+                    "description": "The name of the institution, the first-level location unit, this (shelf) location belongs to",
+                    "type": "string"
+                  },
+                  "code": {
+                    "description": "Code of the (shelf) location, usually an abbreviation of the name",
+                    "type": "string"
+                  }
+                }
```

Comment:
Temporary location is also a reference to a location record, yet here is presented in the same manner.  From looking at the structure of this raml and the underlying sql producing the view, it seemed that the entire point of this API was to provide a hierarchical representation of the data that was *intentionally dereferenced,* I assume because whatever is using it wants human-readable strings, or for some other reason does not want to have to look up and dereference the data itself?

Reply:
> it seemed that the entire point of this API was to provide a hierarchical representation of the data that was intentionally dereferenced, I assume because whatever is using it wants human-readable strings, or for some other reason does not want to have to look up and dereference the data itself?

That is a reasonable assumption. Please could you confirm that with the owner of the issue.

Reply:
I have posted something in the Jira for this issue asking about this.

Reply:
Erin says the use-case for this is to provide data to external systems, which is why the API provides dereferenced, embedded data.

Reply:
Thank you for confirming this